packages/logs/src/boot/startLogs.ts

```diff
@@ -87,6 +90,11 @@ export function doStartLogs(
     )
   }
   errorObservable.subscribe(reportError)
+  if (isExperimentalFeatureEnabled('forward-logs')) {
+    trackConsoleLogs(['log', 'info', 'debug', 'warn'], (log: ConsoleLog) => {
+      errorLogger[log.status](log.message, { date: log.startClocks.timeStamp })
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏`errorLogger` does not seem appropriate any more

Reply:
Indeed, just `logger` seems better